internal/ethereum/ethereum.go

```diff
@@ -46,22 +46,28 @@ func ConstructTransfer(from string, to string, amount *big.Int) ([]byte, error)
 	fromAddress := parseAddress(from)
 	toAddress := parseAddress(to)
 
+	gasPrice, err := Client.SuggestGasPrice(ctx)
+	if err != nil {
+		return []byte{}, errors.Wrapf(err, "cannot fetch suggested gas price")
+	}
+
+	gasTipCap, err := Client.SuggestGasTipCap(ctx)
+	if err != nil {
+		return []byte{}, errors.Wrapf(err, "cannot fetch suggested gas tip cap")
+	}
+
 	nonce, err := Client.PendingNonceAt(ctx, fromAddress)
 	if err != nil {
 		return []byte{}, errors.Wrapf(err, "cannot fetch nonce for address %s", from)
 	}
 
 	gasLimit := uint64(21000)
-	// maxPriorityFeePerGas = 2 Gwei
-	tip := big.NewInt(2000000000)
-	// maxFeePerGas = 20 Gwei
-	feeCap := big.NewInt(20000000000)
 
 	return messageToSign(types.NewTx(&types.DynamicFeeTx{
 		ChainID:   params.SepoliaChainConfig.ChainID,
 		Nonce:     nonce,
-		GasFeeCap: feeCap,
-		GasTipCap: tip,
+		GasFeeCap: gasPrice,
```

Comment:
Shouldn't this be `(gasPrice * gasLimit)` or `(gasPrice * (gasLimit + gasTipCap))` (if we want to be more exact)?

Reply:
I think Geth does the math for us 😍  -- `GasFeeCap` and `GasTipCap` are values _per gas_

Reply:
ahhhh nice. If that's the case we're all good!